ext/opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -162,9 +192,11 @@ def export(
             metric_descriptor = self._get_metric_descriptor(record)
             if not metric_descriptor:
                 continue
-
-            series = TimeSeries()
-            self._add_resource_info(series)
+            series = TimeSeries(
+                resource=self._get_monitored_resource(
+                    record.instrument.meter.resource
```

Comment:
Unrelated to this PR, is resource being part of `MeterProvider` appropriate? Do you think users would have different resources per `meter`?

Reply:
Resources to me encapsulates only information about the machine the code is running on. That shouldn't change between `meter`.